exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -338,6 +346,15 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
             );
           }
 
+          dispatchDebug({
+            type: 'cacheHit',
+            message: `A requested operation was found and returned from the cache.`,
+            operation: res.operation,
```

Comment:
What parts of operation are we using, just wondering if this is too much

Reply:
I’m concerned it may not always be serialisable, ie fetch options being a function.

Does it make sense to just send over the request (query, variables, key) and then just display the rest from/on the events?

Reply:
We're currently [serialising the dispatched events](https://github.com/FormidableLabs/urql-devtools-exchange/pull/26/files#diff-76d6bc8407250594abb3ec8eb8a74cf8R97) in the devtools exchange so I'm not too concerned about this as long as we don't have / introduce circular data structures.

From a DX experience, this just means there's no need to destructure the operation every dispatch and gives us more flexibility down the road.

Reply:
Ok; wasn’t too sure how this would be hooked up but sounds good to me 👍